File: SellerDroidSolution/SellerDroidWeb/Models/DTO/ProductSummary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SellerDroidWeb.Models.DTO
{
    public class ProductSummary : Entity
    {
        public double averageRating { get; set; }
        public double currentUserRating { get; set; }
    }
}

File: SellerDroidSolution/SellerDroid/app/src/main/java/lb/edu/aust/sellerdroid/dto/ProductSummary.java
package lb.edu.aust.sellerdroid.dto;

/**
 * Created by AUST Student on 19/12/2015.
 */
public class ProductSummary extends Entity {

    public float averageRating = 2.5f;
    public float currentUserRating = -1;

    public ProductSummary() {}

    public ProductSummary(String theName, String theDescription) {
        this.name = theName;
        this.description = theDescription;
    }
}
